.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug Report
+about: Report errors and problems
+
+---
+
+<!--
+    The Symfony Code of Conduct applies to all the activity on this repository.
+    See https://symfony.com/doc/current/contributing/code_of_conduct/index.html
+-->
```

Comment:
Sadly I don't understand your comment. By the way, we recently changed the CoC section a bit so we have a dedicated page to show the contents of the CoC (https://github.com/symfony/symfony-docs/pull/9725) instead of displaying them in the index page of the CoC section. So, we could link to the CoC contents directly if we want ... or maybe it's better to keep linking to the index page of the CoC?